single_kernel_mongo/managers/ldap.py

```diff
@@ -217,7 +221,7 @@ def get_ldap_connection_status(self) -> StatusBase:
                 tls = LDAPTls(
                     validate=ssl.CERT_REQUIRED,
                     version=ssl.PROTOCOL_TLSv1_2,
-                    ca_certs_file=f"{self.state.paths.ldap_certificates_file}",
+                    ca_certs_data="\n".join(cert_chain),
```

Comment:
I saw in a sepearate PR you had a `chain ` property that concatenates them together, I assume this will be merged her e

Reply:
I have updated the chain property to be consistent: it takes a list, and returns a list so that we always handle the chain as a list of certificates instead of a concatenated string of certificates.
There were multiple cases where it was painful to keep a big string while keeping the list is more reliable and more consistent with what it represents.